### PR TITLE
Fix spacing between pricing fields

### LINKED ISSUE #38097: Product Block Editor: Pricing section has inconsistent spacing between fields
**Description**

We have some inconsistent spacing across the Product Block Editor, this is particularly noticeable within the pricing section.
This is likely due to some fields being separate blocks, while others are contained within the same block.

**Acceptance criteria**

- [ ] Address spacing within pricing tab to match that of the Figma wireframe.

### PR DESCRIPTION
**Pull request by mdperez86**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #38097

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Got to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
2. Under Features tab make sure to enable `product-block-editor`
3. Then visit `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. Under `Pricing` tab and within `Pricing` section the spacing between the List price field and the Schedule sale should be as described in AkVNGImLgSqCObTQ3idVn7-fi-305-38548

### CODE REVIEW

File: packages/js/product-editor/src/blocks/schedule-sale/editor.scss [hunk 1]
```diff
@@ -1,6 +1,18 @@
 .wp-block-woocommerce-product-schedule-sale-fields {
+	.components-toggle-control {
+		margin-bottom: $gap-large;
```
Comment (joshuatf) [inline packages/js/product-editor/src/blocks/schedule-sale/editor.scss]: I'm seeing this as 32px in the design:

Screen Shot 2023-05-05 at 11 48 17 AM

**Review by joshuatf — COMMENTED** [review]
Thanks for making these improvements!  Looking great; just left one minor dimension change suggestion I'm seeing in the design.

**Review by joshuatf — APPROVED** [review]
Thanks for the changes, Maikel!  This LGTM!

**Comment by rodelgc** [comment]
Uploading expected design
rlLzUUNHuU

**Pull request merged**